Configuration/PyReleaseValidation/python/relval_extendedgen.py

```diff
@@ -41,6 +41,8 @@
 workflows[519]=['WTolNuJets_LO_Mad_13TeV_py8_taurhonu',['WTolNu01234Jets_5f_LO_MLM_Madgraph_LHE_13TeV','Hadronizer_TuneCUETP8M1_13TeV_MLM_5f_max4j_LHE_pythia8_taurhonu','HARVESTGEN2']]
 workflows[520]=['VHToHtt_NLO_Pow_13TeV_py8_taurhonu',['VHToH_Pow_LHE_13TeV','Hadronizer_TuneCUETP8M1_13TeV_Httrhonu_powhegEmissionVeto_pythia8','HARVESTGEN2']]
 
+workflows[535] = ['', ['TTbar_13TeV_Pow_herwig7','HARVESTGEN']]
```

Comment:
This workflow is not in the master. What is the plan for master?

Reply:
Hi @srimanob 535 is there in master https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_extendedgen.py#L26

Reply:
Thanks. Can it move to the place where it is in master? The code is very confusing to compare now. Order is confusion, and it is annoying when I try to compare with master.

Reply:
I put this at the end of the "LHE Generation + Hadronization" block since this is what this workflow is. I'm not sure why in master it's in a "Matrix Element Generations" block, but when I make a PR to master to add more workflows, I could move it to where this one is now?

Reply:
Thanks, please agree with @cms-sw/generators-l2 and adapt it to the proper place.

Reply:
I agree with LHE + Hadronization. We should fix in master.